services/src/main/java/org/keycloak/keys/Attributes.java

```diff
@@ -55,6 +55,14 @@ public interface Attributes {
     ProviderConfigProperty KEY_USE_PROPERTY = new ProviderConfigProperty(KEY_USE, "Key use", "Whether the key should be used for signing or encryption.", LIST_TYPE,
             KeyUse.SIG.getSpecName(), KeyUse.SIG.getSpecName(), KeyUse.ENC.getSpecName());
 
+    String EC_GENERATE_CERTIFICATE_KEY = "ecGenerateCertificate";
+    ProviderConfigProperty EC_GENERATE_CERTIFICATE_PROPERTY = new ProviderConfigProperty(
+            EC_GENERATE_CERTIFICATE_KEY,
+            "Generate Certificate",
+            "If a certificate should be build on creation",
```

Comment:
```suggestion
            "If a certificate should be build on creation. If the certificate is build, it will be available in the realm JWK for the key in the claim x5c and corresponding thumbprints may be available in the claims like x5t or x5t#S256",
```
Is it possible to add a little bit more details, so it is more clear what the config option is about even for someone not familiar with this? Does the text I suggested makes sense to you?

Reply:
I think your text is good. I will use it as you have provided it